test/config.test.ts

```diff
@@ -823,6 +873,31 @@ describe('readConfig', () => {
     expect(importModule).toHaveBeenCalledWith('config.js');
   });
 
+  it('should attempt to find a js config file given a directory path', async () => {
+    const mockConfig = {
+      description: 'Test config',
+      providers: ['openai:gpt-4o'],
+      prompts: ['Hello, world!'],
+    };
+    jest.spyOn(fs, 'readFileSync').mockReturnValue(yaml.dump(mockConfig));
+    jest.mocked(importModule).mockResolvedValue(mockConfig);
+    jest.spyOn(fs, 'accessSync').mockImplementationOnce(() => {
+      throw new Error('File does not exist')
+    });
+    jest.spyOn(fs, 'accessSync').mockImplementationOnce(() => {
+      throw new Error('File does not exist')
+    });
+    jest.spyOn(fs, 'accessSync').mockImplementationOnce(() => {
+      throw new Error('File does not exist')
+    });
+    
+
+    const result = await readConfig('configs');
+
+    expect(result).toEqual(mockConfig);
+    expect(importModule).toHaveBeenCalledWith(path.normalize('configs/promptfooconfig.js'));
```

Comment:
Using `path.normalize` here to transform the url path specifically for testing with node in windows environments.